quickstarts/oma-aqm/config.yml

```diff
@@ -0,0 +1,44 @@
+# Name of the quickstart (required)
+name: alert-quality-management
+
+# Displayed in the UI (required)
+title: Alert Quality Management Quickstart
+
+# Long-form description of the quickstart (required)
+description: |
+  Alert Quality Management (AQM) focuses on reducing the number of nuisance incidents so that you focus only on alerts with true business impact.  This reduces alert fatigue and ensures that you and your team focus your attention on the right places at the right times.
+  
+  For more information, see https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide/
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Reduce the volume of nuisance incidents with the Alert Quality Management dashboard and process.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
```

Comment:
by setting this to community, it means it's not an official New Relic quickstart and was built by 3rd party creators, I just want to make sure is the correct level to set this to.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#levels